app/javascript/tooltips.js

```diff
@@ -2,7 +2,7 @@ import tippy from "tippy.js"
 
 document.addEventListener('turbolinks:load', function() {
   tippy('.gold-split', {placement: 'left'})
-  tippy('.tip', {placement: 'right'})
+  tippy('.tip', {placement: 'top'})
```

Comment:
What's the purpose of having both `.tip` and `.tip-top`?

Reply:
`.tip` should be the default, like "I don't care where the tooltip is positioned, just use whatever is consistent with the rest of the site". Right now that's `top`, but maybe later we change it.

Whereas `.tip-top` is for "I specifically want the tooltip to be above the text", which could be the case if there's blank space above the text but not left, right, or below it.